server/deskroom/tags/endpoints.py

```diff
@@ -15,28 +15,32 @@
 
 @router.put("/", response_model=UpdateTagsOut)
 async def update_tags(
-    update_request: UpdateTagsIn,
+    request: UpdateTagsIn,
     supabase: AsyncClient = Depends(create_supabase_async_client),
     current_user: DeskroomUser = Depends(get_current_user),
 ) -> dict[str, Any]:
-    knowledge_tags = update_request.knowledge_tags
-    category_id = update_request.category_id
-    knowledge_base_id = update_request.knowledge_base_id
-    organization_key = update_request.organization_key
+    category_id = request.category_id
```

Comment:
UpdateTagsIn 이라는 DTO가 여기서만 사용한다면 endpoint layer의 복잡한 로직을 DTO class 안으로 옮길 수 있지 않을까요? 

```suggestion
class UpdateTagsIn(Schema):
    ...
    def get_insert_payload(self):
        return {"category_id": self.category_id, "question_id": self.knowledge_base_id}
```